src/Console/EasyCodingStandardConsoleApplication.php

```diff
@@ -56,6 +58,14 @@ public function doRun(InputInterface $input, OutputInterface $output): int
             unset($xdebugHandler);
         }
 
+        if ($input->hasParameterOption(['--version', '-V'], true)) {
+            $output->writeln($this->getLongVersion());
+            $output->writeln(sprintf('+ %s <info>%s</info>', 'PHP_CodeSniffer', PHP_CodeSniffer::VERSION));
+            $output->writeln(sprintf('+ %s <info>%s</info>', 'PHP-CS-Fixer', PhpCsFixer::VERSION));
+
+            return 0;
```

Comment:
is the `shouldPrintMetaInformation()` no longer reachable? when it executed?

Reply:
It's unclear what kind of pattern the original code was intended for, but this pattern certainly doesn't exist, so I removed it.
Maybe you were expecting an option like `./bin/ecs --output-format=console --version`, but I don't understand your use case.